rules/zoom_operation_rules/zoom_new_meeting_passcode_required_disabled.yml

```diff
@@ -4,6 +4,7 @@ DisplayName: "Zoom New Meeting Passcode Required Disabled"
 Enabled: true
 Filename: zoom_new_meeting_passcode_required_disabled.py
 Runbook: Confirm this user acted with valid business intent and determine whether this activity was authorized.
+Reference: https://support.zoom.com/hc/en/article?id=zm_kb&sysparm_article=KB0063160#:~:text=Since%20September%202022%2C%20Zoom%20requires,enforced%20for%20all%20free%20accounts.
```

Comment:
```suggestion
Reference: https://support.zoom.com/hc/en/article?id=zm_kb&sysparm_article=KB0063160#:~:text=Since%20September%202022%2C%20Zoom%20requires,enforced%20for%20all%20free%20accounts
```